string front_marker_color
string back_marker_color
string name
